Library/Formula/capnp.rb

```diff
@@ -14,6 +14,7 @@ class Capnp < Formula
   option "without-shared", "Disable building shared library variant"
 
   def install
+    ENV.cxx11 if MacOS.version < :mavericks
```

Comment:
I'd guess a `needs :cxx11` may be needed too. Also, that should maybe be optional (or at least recommended). Can you confirm this fixes the issue or is it a guess?